client/components/QandA/QAComponents/AnswerEntry.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+const AnswerEntry = ({ answer }) => {
+  // line 5, prints single answer
+ // console.log('this is answers ', answer[0][1])
+  if (!answer[0]) {
+    return null;
+  } else {
+    return (
+      <div id="AnswerEntry">
+        <div>
+        Answer: {answer[0][1].body}
```

Comment:
For the sake of conciseness maybe assign this to a variable. like answer = answer[0][1]